make/kind.mk

```diff
@@ -7,10 +7,10 @@ KIND_CLUSTER_NAME ?= $(GITHUB_REPOSITORY)-dev
 KIND_KUBECONFIG ?= $(KIND_DIR)/$(KIND_CLUSTER_NAME)/kubeconfig
 
 KINDEST_NODE_IMAGE ?= ghcr.io/mesosphere/kind-node
-KINDEST_NODE_VERSION_v1.27 ?= v1.27.13
-KINDEST_NODE_VERSION_v1.28 ?= v1.28.9
-KINDEST_NODE_VERSION_v1.29 ?= v1.29.4
-KINDEST_NODE_VERSION_v1.30 ?= v1.30.0
+KINDEST_NODE_VERSION_v1.27 ?= v1.27.15
+KINDEST_NODE_VERSION_v1.28 ?= v1.28.11
+KINDEST_NODE_VERSION_v1.29 ?= v1.29.6
+KINDEST_NODE_VERSION_v1.30 ?= v1.30.2
```

Comment:
https://hub.docker.com/r/kindest/node/tags?page=&page_size=&ordering=&name=1.29
please ensure these tags are actually available e.g. 1.29.6 is not

Reply:
neither is 1.30.2, 1.28.11, or 1.27.15

Reply:
nvm, I wasn't aware of https://github.com/mesosphere/kind-docker-image-automation